scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -2,9 +2,13 @@
 
 # fail if some commands fails
 set -e
+
+# Do not show token in CI log
+set +x
+export GITHUB_TOKEN=`cat $KAM_GITHUB_TOKEN_FILE`
```

Comment:
Where is `KAM_GITHUB_TOKEN_FILE`defined?

Reply:
https://github.com/openshift/release/blob/master/ci-operator/config/redhat-developer/kam/redhat-developer-kam-master__v4.5.yaml#L42